plugins/flytekit-papermill/flytekitplugins/papermill/task.py

```diff
@@ -165,13 +166,14 @@ def __init__(
         if not os.path.exists(self._notebook_path):
             raise ValueError(f"Illegal notebook path passed in {self._notebook_path}")
 
-        if outputs:
+        if outputs and output_notebooks:
```

Comment:
```suggestion
        if output_notebooks:
```

Reply:
what if you just want a notebook?  don't want to have to make a fake output.

Reply:
updated it